omero/sysadmins/version-requirements.rst

```diff
@@ -259,62 +213,22 @@ OMERO support policies
     * - Version
       - Release date
       - Supported until
-      - OMERO 5.4
-      - OMERO 5.5
       - OMERO 5.6
-    * - 9.3
-      - Sep 2013
-      - Sep 2018
-      - |Deprecated|
-      - |Dropped|
-      - |Dropped|
-    * - 9.4
-      - Dec 2014
-      - Dec 2019
-      - |Supported|
-      - |Deprecated|
-      - |Dropped|
-    * - 9.5
-      - Jan 2016
-      - Jan 2021
-      - |Supported|
-      - |Supported|
-      - |Deprecated|
-    * - 9.6
-      - Sep 2016
-      - Sep 2021
-      - |Recommended|
-      - |Supported|
-      - |Deprecated|
-    * - 10
-      - Oct 2017
-      - Nov 2022
-      - |Upcoming|
-      - |Recommended|
-      - |Deprecated|
-    * - 11
-      - Oct 2018
-      - Nov 2023
-      - |Upcoming|
-      - |Upcoming|
-      - |Recommended|
     * - 12
       - Sep 2019
       - Nov 2024
-      - |Upcoming|
-      - |Upcoming|
       - |Supported|
     * - 13
       - Sep 2020
       - Nov 2025
-      - |Upcoming|
-      - |Upcoming|
-      - |Supported|
+      - |Recommended|
     * - 14
       - Sep 2021
       - Nov 2026
-      - |Upcoming|
-      - |Upcoming|
+      - |Supported|
```

Comment:
Proposing to use this as `|Recommended|` as this is the default PSQL version coming with Ubuntu 22.04